pants-plugins/internal_plugins/test_lockfile_fixtures/BUILD

```diff
@@ -0,0 +1,4 @@
+# Copyright 2022 Pants project contributors (see CONTRIBUTORS.md).
```

Comment:
Thoughts on calling this `jvm_test_lockfile_fixtures`?

Reply:
We'd probably add Python test lockfiles, right?

Reply:
Makes sense. And yeah we'd want to use the same goal etc.

This is entirely internal also, so not really a big deal either way to change later